package.json

```diff
@@ -12,27 +12,25 @@
   "author": "sebworks",
   "private": true,
   "scripts": {
-    "dev": "pnpm install && pnpm i18n:create-locales-list && pnpm i18n:update-locales && cross-env nuxt --hostname 0.0.0.0",
+    "dev": "pnpm install && pnpm i18n:create-locales-list && pnpm i18n:update-locales && nuxt --hostname 0.0.0.0",
```

Comment:
Thank you for removing unnecessary `cross-env`! I did notice it and think of removing it a couple of times, but got distracted every time...